LEGO1/LegoOmni.mingw.def

```diff
@@ -0,0 +1,172 @@
+; LegoOmni.def : Declares the module paarameters for the LEGO1DLL.
+
+DESCRIPTION  " Lego OMNI Windows Dynamic Link Library"
+
+EXPORTS
+
+; EXPORTs really required only.
+
+_ZN19LegoBackgroundColorC1EPKcS1_@8 = _ZN19LegoBackgroundColorC1EPKcS1_
```

Comment:
I generated this file using mingw64's gendef script, and then fixed up these stdcall functions.
I think this list can be pruned to (approximately) the same size as the msvc one, but that's a job for another time :)